### PR TITLE
Add Command#getVersion

### PR DESCRIPTION
**Pull request by Xirado**
[contributing]: https://github.com/DV8FromTheWorld/JDA/wiki/5%29-Contributing

## Pull Request Etiquette

- [x] I have checked the PRs for upcoming features/bug fixes.
- [x] I have read the [contributing guidelines][contributing].

### Changes

- [x] Internal code
- [x] Library interface (affecting end-user code) 
- [x] Documentation
- [ ] Other: \_____ 

Closes Issue: NaN

## Description

This PR adds Command#getVersion. ~(Not yet documented by Discord)~ **UPDATE**: [PR Merged](https://github.com/discord/discord-api-docs/pull/3524)

### CODE REVIEW

**Comment by RedDaedalus** [comment]
Since this field is undocumented (and I believe will not be documented) it could be changed or removed at any time.

**Comment by Xirado** [comment]
> Since this field is undocumented (and I believe will not be documented) it could be changed or removed at any time.

@RedDaedalus Yes it could be changed or removed at any time. This is just a placeholder atm and i'm waiting for it to get documented. Minn told me it was fine to make this PR but we'll wait

File: src/main/java/net/dv8tion/jda/api/interactions/commands/Command.java
Comment (DV8FromTheWorld) [inline src/main/java/net/dv8tion/jda/api/interactions/commands/Command.java:322]:
```suggestion
     * @return The version of the command as a snowflake id.
```

File: src/main/java/net/dv8tion/jda/api/interactions/commands/Command.java
Comment (DV8FromTheWorld) [inline src/main/java/net/dv8tion/jda/api/interactions/commands/Command.java:323]:
```suggestion
     *
     * @see #getTimeModified()
     */
```

**Comment by DV8FromTheWorld** [comment]
On hold until docs PR is merged.
If waiting goes longer than 9/1/2021, feel free to bump.

File: src/main/java/net/dv8tion/jda/api/interactions/commands/Command.java
Comment (MinnDevelopment) [inline src/main/java/net/dv8tion/jda/api/interactions/commands/Command.java:80]: Just in case, lets use the id as a fallback:

```suggestion
        this.version = json.getUnsignedLong("version", id);
```

**Review by DV8FromTheWorld — APPROVED** [review]

File: src/main/java/net/dv8tion/jda/api/interactions/commands/Command.java
Comment (Xirado) [reply src/main/java/net/dv8tion/jda/api/interactions/commands/Command.java:80]: Too late

**Pull request merged**
